### ISSUE TITLE
ERROR connmgr: tried to remove tag from untracked peer: <peer.ID > connmgr.go:154

### ISSUE THREAD

**Issue by leerspace** [ipfs/go-ipfs#4353]
#### Version information:
go-ipfs version: 0.4.12-rc1-
Repo version: 6
System version: amd64/linux
Golang version: go1.9

#### Type:
Bug

#### Severity:
Medium

#### Description:
The IPFS daemon outputs a lot of these errors for different peers, but it's unclear what the problem is. In some cases the peer ID appears to be null.

```
02:22:18.862 ERROR    connmgr: tried to remove tag from untracked peer: <peer.ID > connmgr.go:154
02:22:32.128 ERROR    connmgr: tried to remove tag from untracked peer: <peer.ID ZS3tbT> connmgr.go:154
02:22:54.964 ERROR    connmgr: tried to remove tag from untracked peer: <peer.ID > connmgr.go:154
02:22:56.619 ERROR    connmgr: tried to remove tag from untracked peer: <peer.ID > connmgr.go:154
02:24:52.132 ERROR    connmgr: tried to remove tag from untracked peer: <peer.ID PTQaNw> connmgr.go:154
02:24:52.132 ERROR    connmgr: tried to remove tag from untracked peer: <peer.ID a5nPLj> connmgr.go:154
02:25:07.132 ERROR    connmgr: tried to remove tag from untracked peer: <peer.ID SzdURu> connmgr.go:154
02:26:22.135 ERROR    connmgr: tried to remove tag from untracked peer: <peer.ID d6Nq8y> connmgr.go:154
02:27:02.137 ERROR    connmgr: tried to remove tag from untracked peer: <peer.ID YwTVux> connmgr.go:154
02:27:32.138 ERROR    connmgr: tried to remove tag from untracked peer: <peer.ID avVMdV> connmgr.go:154
02:28:00.682 ERROR    connmgr: tried to remove tag from untracked peer: <peer.ID dKXzwT> connmgr.go:154
02:28:07.139 ERROR    connmgr: tried to remove tag from untracked peer: <peer.ID avVMdV> connmgr.go:154
02:28:10.550 ERROR    connmgr: tried to remove tag from untracked peer: <peer.ID ZQDeem> connmgr.go:154
02:28:17.128 ERROR    connmgr: tried to remove tag from untracked peer: <peer.ID UZPXfY> connmgr.go:154
```

**Comment by Stebalien** [comment]
That's a spurious error, we should reduce it to a info (or remove it). Thanks for reporting.

Explanation: The connmgr is responsible for tracking how useful our peers are and disconnecting from the less-useful ones when we have too many connections. In this case, we've disconnected from a peer and all the services that use that peer are telling the connmgr that the peer is no longer useful (they're not connected to it anymore).

**Comment by Stebalien** [comment]
Actually, it appears like this has already been fixed in #4349.
